software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java

```diff
@@ -594,4 +595,16 @@ private void addRiakOnPath(ScriptHelper scriptHelper) {
 //        log.warn("riak command not found on PATH. Altering future commands' environment variables from {} to {}", getShellEnvironment(), newPathVariable);
         scriptHelper.environmentVariablesReset(newPathVariable);
     }
+
+    @Override
+    public Integer getNumberOfOpenFiles() {
+        Integer defaultNumberOfFiles = RiakNode.RIAK_NUM_OPEN_FILES.getConfigKey().getDefaultValue();
+        Integer numberOfFiles = entity.getConfig(RiakNode.RIAK_NUM_OPEN_FILES);
+        if (numberOfFiles.compareTo(defaultNumberOfFiles) < 0) {
+            log.warn("Specified number of open files : {} : less than the required minimum. Using defaults : {}.", numberOfFiles, defaultNumberOfFiles);
+            entity.setAttribute(RiakNode.RIAK_NUM_OPEN_FILES, defaultNumberOfFiles);
```

Comment:
Completely agree. It would be better to notify the user and stop the installation.